bot/karma_command.go

```diff
@@ -53,7 +53,8 @@ func NewKarmaCommand(store db.Store, w io.Writer) cli.Command {
 			for i := 0; i < c.Int("count") && i < len(keys); i++ {
 				karma := results[keys[i]]
 				text := fmt.Sprintf("karma for *%s*: %s\n", keys[i], karma)
-				if _, err := w.Write([]byte(text)); err != nil {
+
+				if err := write(w, text); err != nil {
```

Comment:
let's change this implementation a bit - lets just build one long string, e.g. 
```
text := ...
text += ...
```
and only call `write` once at the bottom (like we do for all the other commands)